nix/scripts/node_modules.sh

```diff
@@ -45,7 +45,10 @@ copyNodeModules() {
 
   # WARNING: We can't de-reference .bin symlinks
   cp -Rf ${src}/node_modules/.bin/. "${tmp}/.bin/"
-  rm -r "${dst}"
+  # we need to rm -rf after upgrading react-native-reanimated to 3.3.0
+  # they use debug flags in Cmake Android project when the app is built in debug mode
+  # those files are right protected by default
+  rm -rf "${dst}"
```

Comment:
Wouldn't it make more sense to just to `chmod` the problematic folder?

Reply:
I did that, however I notice an occasionally harmless `chmod` warning
`chmod: cannot access 'node_modules/react-native-reanimated/android/.cxx': No such file or directory`
 and it maybe confusing for others who do not know the context...

Reply:
You can just add `--silent`/`-f`.